Wikipedia/Code/WMFNotificationsController.m

```diff
@@ -162,70 +162,69 @@ - (void)sendNotificationWithTitle:(NSString *)title body:(NSString *)body catego
+                                                    //                                                            CGFloat normalizedHeight = WMFNotificationImageCropNormalizedMinDimension * aspect;
+                                                    //                                                            CGFloat halfNormalizedHeight = 0.5 * normalizedHeight;
+                                                    //                                                            CGFloat originY = MAX(0, faceMidY - halfNormalizedHeight);
+                                                    //                                                            CGFloat normalizedWidth = MAX(faceRect.size.width, WMFNotificationImageCropNormalizedMinDimension);
+                                                    //                                                            CGFloat originX = 0.5 * (1 - normalizedWidth);
+                                                    //                                                            cropRect = CGRectMake(originX, originY, normalizedWidth, normalizedHeight);
+                                                    //                                                        } else {
+                                                    //                                                            CGFloat faceMidX = CGRectGetMidX(faceRect);
+                                                    //                                                            CGFloat normalizedWidth = WMFNotificationImageCropNormalizedMinDimension / aspect;
+                                                    //                                                            CGFloat halfNormalizedWidth = 0.5 * normalizedWidth;
+                                                    //                                                            CGFloat originX = MAX(0, faceMidX - halfNormalizedWidth);
+                                                    //                                                            CGFloat normalizedHeight = MAX(faceRect.size.height, WMFNotificationImageCropNormalizedMinDimension);
+                                                    //                                                            CGFloat originY = 0.5 * (1 - normalizedHeight);
+                                                    //                                                            cropRect = CGRectMake(originX, originY, normalizedWidth, normalizedHeight);
+                                                    //                                                        }
+                                                    //                                                    }
+                                                    
+                                                    //Since face cropping is broken, don't attach images with faces
```

Comment:
@amrox broken specifically in the notifications (not on our end) - the content rects I passed weren't respected by the system notification display. this code wasn't added, just updated to use the new image loading